ccmlib/scylla_node.py

```diff
@@ -1785,6 +1785,78 @@ def wait_for_compactions(self,
                                      column_family=column_family,
                                      idle_timeout=idle_timeout)
 
+    def run_scylla_types(self, action: str, values: List[str], types: List[str], additional_args: Optional[List[str]]=None, text=True):
+        """Invoke scylla-types command, with the specified action and additional_args.
+
+        For more information about scylla-types, see https://opensource.docs.scylladb.com/stable/operating-scylla/admin-tools/scylla-types.html
+
+        Params:
+        * action - The scylla-types operation to run.
+        * additional_args - Additional command-line arguments to pass to scylla-types action, this should be a list of strings.
+        * text - If True, output of the command is treated as text, if not as bytes
+
+        Returns: stdout, stderr
+
+        Raises: subprocess.CalledProcessError if scylla-types returns a non-zero exit code.
+        """
+        if additional_args is None:
+            additional_args = []
+
+        for value_type in types:
+            additional_args.extend(["-t", value_type])
+
+        scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
+        scylla_types_cmd_with_args = [scylla_path, "types", action] + additional_args + ["--"] + values
+        res = subprocess.run(scylla_types_cmd_with_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=text, check=False, env=self._get_environ())
+        if res.returncode:
+            raise ToolError(command=' '.join(scylla_types_cmd_with_args), exit_status=res.returncode, stdout=res.stdout, stderr=res.stderr)
+        return (res.stdout, res.stderr)
+
+    def serialize_values(self, values: List[str], types: List[str], prefixable_compounds=False) -> str:
```

Comment:
I'd prefer if we had just `run_scylla_types()`, instead of exposing a separate methods for the different actions, this will cause an explosion of methods.

Reply:
Done.